pkg/node/node.go

```diff
@@ -258,7 +258,6 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 		return nil, fmt.Errorf("localstore: %w", err)
 	}
 	b.localstoreCloser = storer
```

Comment:
there needs to be a change here. setting up the batchstore with localstore.Unreserve hood